src/components/WishList/index.js

```diff
@@ -0,0 +1,102 @@
+import React, { Component } from 'react';
+import Property from './Property';
```

Comment:
See we don't want duplicate code, just make a shared folder and put this file in that , then import it in this file and also in search file from that file